File: src/test/java/org/jdc/template/inject/DbBlankTestModule.java
package org.jdc.template.inject;

import org.dbtools.android.domain.config.DatabaseConfig;
import org.jdc.template.model.database.TestMainDatabaseConfig;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class DbBlankTestModule {
    @Provides
    @Singleton
    DatabaseConfig provideDatabaseConfig() {
        return TestMainDatabaseConfig.getInstance();
    }
}

File: src/test/kotlin/org/jdc/template/inject/DbBlankTestModule.kt
package org.jdc.template.inject

import dagger.Module
import dagger.Provides
import org.dbtools.android.domain.config.DatabaseConfig
import org.jdc.template.model.database.TestMainDatabaseConfig
import javax.inject.Singleton

@Module
class DbBlankTestModule {
    @Provides
    @Singleton
    internal fun provideDatabaseConfig(): DatabaseConfig {
        return TestMainDatabaseConfig.instance
    }
}
